conduct/COC_POLICY.md

```diff
@@ -0,0 +1,128 @@
+
+The CoC Team is empowered to involve external people to resolve an issue provided these commit to abide by the code of conduct, the [incident management processes][], and by this policy.
+
+### Delegation to projects
+
+Projects may opt-in to enforce the code of conduct themselves provided they meet the following requirements:
+
+1. Commit to uphold the code of conduct.
+2. Commit to uphold this policy.
+3. Provide a general email address that incidents can be reported to and document it alongside the code of conduct.
+4. Define and publicly document a process by which the project selects people responsible for managing incident reports. For Impact and At Large projects, at least two people must be selected.
+5. List the people responsible for managing incident reports alongside the code of conduct as outlined in the [Reporting incidents section](#reporting-incidents).
+6. Publicly document their decision-making process for responding to incident reports.
+7. Document the foundation's escalation and appeal process alongside the code of conduct.
+8. Participate in good faith in the [escalation](#escalation) and [appeal](#appeals) processes.
+
+## Moderation
+
+Each project is free to implement moderation processes appropriate for the size and scope of the project provided these processes abide by the code of conduct and by this policy.
+
+## Escalation
+
+Reporters, harmed individuals, and accused individuals may [escalate](#escalation) an issue to the CoC Team at any point in the process in case of a conflict of interest, if the report is not being handled appropriately (for example, if there are unacceptable delays, a lack of communication, confidentiality isn't respected, etc.), or in case of an emergency.
+
+The CoC Team is empowered to manage escalation requests of incident reports managed by projects that have opted-in to enforce the code of conduct themselves.
+
+If the CoC Team finds that the escalation request has merit, the CoC Team may decide to takeover the management of the incident report or refer the issue to the Cross Project Council which may request changes to the existing management process. 
+
+Projects are expected to participate in good faith in the escalation process and to share all the information related to the incident report with the CoC Team. The CoC Team is expected to consider project and community needs during the escalation process and may involve the Cross Project Council if necessary and project leadership if appropriate.
+
+## Appeals
+
+Harmed individuals and accused individuals have thirty (30) days to appeal to the CoC Team a decision made in response to an incident report. Decisions may only be appealled on the grounds that the documented process was not properly followed or that it did not meet the requirements of this policy. If the CoC Team finds that the appeal has merit, it may decide to override the decision that was previously made.
```

Comment:
```suggestion
Harmed individuals and accused individuals have thirty (30) days after a response was provided to appeal to the CoC Team a decision made in response to an incident report. Decisions may only be appealed on the grounds that the documented process was not properly followed or that it did not meet the requirements of this policy. If the CoC Team finds that the appeal has merit, it may decide to override the decision that was previously made.
```